agent_event_publisher_http/README.md

```diff
@@ -19,3 +20,53 @@ connection: {
   ]
 }
 ```
+
+### Available events
+
+#### `credential`
+
+```
+UnsignedCredentialCreated
+SignedCredentialCreated
+CredentialSigned
+```
+
+#### `offer`
+
+```
+CredentialOfferCreated
+CredentialAdded
+FormUrlEncodedCredentialOfferCreated
+TokenResponseCreated
+CredentialRequestVerified
+CredentialResponseCreated
+```
+
+#### `server_config`
+
+```
+ServerMetadataLoaded
+CredentialsSupportedCreated
+```
+
+#### `agent_secret_manager`
+
+```
+Initialized
+DidMethodEnabled
+```
```

Comment:
These can not be published because the `AgentSecretManager` aggregate is not part of neither the `IssuanceState` nor the `VerificationState`. 

The `Aggregate` trait is indeed implemented for `AgentSecretManager`, but nothing else CQRS is applied to it. Which means that all related events are not persisted in the state and therefore the Event Publisher can not 'listen' to them (and thus won't be able to publish these events). 

The only Aggregates + Events that are being persisted and able to be published are those defined in `agent_issuance` and `agent_verification`: https://github.com/impierce/ssi-agent/blob/dev/agent_store/src/postgres.rs